src/vitabel/vitals.py

```diff
@@ -314,6 +314,32 @@ def add_defibrillator_recording(self, filepath: Path | str, metadata={}) -> None
                     self.data.add_channel(chan)
 
         self.metadata["Recording_files_added"].append(str(filepath))
+    
+    def add_ventilatory_feedback(
```

Comment:
I think it would be a good idea to be a bit more explicit here and rename this:

```suggestion
    def add_eolife_ventilatory_feedback(
```

Reply:
Following an offline discussion, we keep it as it is.